akd/src/storage/types.rs

```diff
@@ -20,8 +19,8 @@ pub enum StorageType {
     Azks = 1,
     /// HistoryTreeNode
     HistoryTreeNode = 2,
-    /// HistoryNodeState
-    HistoryNodeState = 3,
+    /// EOZ: HistoryNodeState = 3 was removed from here.
```

Comment:
To answer the question in the comment: We're going to do a DB break anyways, we can update this to 3. If we were adding types and trying for a db upgrade, then yeah. But until we release "version 1" we don't have much to worry about if we're going to break all backwards compat.